python/lsst/obs/decam/decamMapper.py

```diff
@@ -242,12 +240,12 @@ def std_raw(self, item, dataId):
         # from the metadata. Once TPV is supported, the following keyword
         # removal may not be necessary here and would probably be done at
```

Comment:
May you please correct these comments too? These comments were there partially because of the line you removed. My understanding is the previous behavior of `afw.image.makeWcs` would not modify the input metadata at all if it's PTV.  And the behavior has changed now with DM-9298.  Now it removes some wcs cards  but not PV cards. Is that correct?